rrfs-test/validated_yamls/templates/obtype_config/adpupa_specificHumidity_120.yaml

```diff
@@ -14,7 +14,7 @@
          obsdataout:
            engine:
              type: H5File
-             obsfile: jdiag_adpupa_specificHumidity_120.nc4
+             obsfile: jdiag_adpupa_q120.nc4
```

Comment:
should we change q120 to q_120? Also, do we have a better short way for specificHumidity? Moisture has several units. It is better to make it clear which unit is used here.

Reply:
I still prefer using the longer name `jdiag_adpupa_specificHumidity_120.nc`. Then we keep the jedi yaml, obs space name, and the jdiag name all consistent. Keeping them aligned should simplify things from an engineering perspective, IMO.

Reply:
> should we change q120 to q_120? Also, do we have a better short way for specificHumidity? Moisture has several units. It is better to make it clear which unit is used here.

Could you elaborate on what you mean by "better short way for specific humidity"?

Reply:
If you check this line 59 of this page:
https://github.com/NOAA-EMC/rrfs-workflow/pull/661/files
`specific_humidity` has been changed to `water_vapor_mixing_ratio_wrt_moist_air`

I think at the workflow level, using `q120` is far enough, `adpupa_q120` already overkills.